src/components/transactions/TxSummary/styles.module.css

```diff
@@ -46,7 +48,7 @@
   color: var(--color-text-secondary);
 }
 
-@media (max-width: 1460px) {
+@media (max-width: 1350px) {
```

Comment:
Previously we had 1200px here which is a default MUI breakpoint. We should try to stick to that as much as possible so the whole UI changes layout at the same viewports.

Reply:
In this case there is no way to fit everything in one line with 1200px without the execute button at the end getting cut off

Reply:
that is assuming all fields are as wide as they can potentially be

Reply:
I'm for merging this as is @usame-algan and opening a new issue for 1200px. This is good enough and @liliya-soroka confirmed that it is fine on her 13 inch macbook.